src/components/Form/SetAclPermissionForm.jsx

```diff
@@ -115,21 +127,34 @@ const SetAclPermissionForm = () => {
             placeholder={selectedUser}
           />
         </div>
-        <div style={formRowStyle}>
-          <label htmlFor="set-acl-doctype">Select document type: </label>
-          <DocumentSelection htmlId="set-acl-doctype" />{' '}
-        </div>
+        <FormControl fullWidth autoComplete="off">
```

Comment:
`<FormControl>` is a `<form>` element, not a `<div>` or a `<label>`. Use `<FormControl>` to replace the `<form>`. That means Line 119 to 129 and Line 141 to 147 should be inside `<FormControl>`.